vql/linux/tcpsnoop/ebpf.go

```diff
@@ -0,0 +1,109 @@
+//go:build linux
```

Comment:
This also needs to be removed as it's overriding the linux AND linuxbpf from the below line.